### PR TITLE
Issue 16

### PR DESCRIPTION
**Pull request by bealearnscode**
I've added created a handheld device markdown file and added four links. I've also added my name to the list of contributors.

### CODE REVIEW

File: Knowledge_Base/01handheld_device_apps.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[If me - is a community for mental health experiences](https://www.if-me.org/)<br />
```
Comment (andy5995) [inline Knowledge_Base/01handheld_device_apps.md]: Does If Me offer an app for a smartphone or other handheld device?

**Comment by bealearnscode** [comment]
So sorry, it doesn't. I will change it right now.

File: Knowledge_Base/01handheld_device_apps.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[If me - is a community for mental health experiences](https://www.if-me.org/)<br />
+"tags": ["open source", "multiple languages", "resources", "community"]<br />
+
+
+[noCD is a smartphone and smartwatch app bringing technology into OCD treatment](http://www.treatmyocd.com/)<br />
+"tags": ["OCD", "smartphone app", "exposure therapy", "health tech"]<br />
+
```
Comment (andy5995) [inline Knowledge_Base/01handheld_device_apps.md]: need `-<br />` on this line, and L10

File: Knowledge_Base/01handheld_device_apps.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+[noCD is a smartphone and smartwatch app bringing technology into OCD treatment](http://www.treatmyocd.com/)<br />
+"tags": ["OCD", "smartphone app", "exposure therapy", "health tech"]<br />
+
+[BetterHelp is an online counseling app with 24/7 availability](https://www.betterhelp.com/)<br />
+"tags": ["online counseling", "smartphone app", "affordable", "health tech", "convenient hours"]<br />
+
+[Headspace is a meditation app](https://www.headspace.com/)<br />
+"tags": ["meditation", "smartphone app", "breathing exercises", "free trial", "health tech"]
```
Comment (andy5995) [inline Knowledge_Base/01handheld_device_apps.md]: You'll need a `<br />` at the end of this line.
and then another `-<br />`

We'll probably be talking soon about the stupid format of these files, lol. Too many brrr's.

File: Knowledge_Base/01handheld_device_apps.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+[If me - is a community for mental health experiences](https://www.if-me.org/)<br />
+"tags": ["open source", "multiple languages", "resources", "community"]<br />
+
+
+[noCD is a smartphone and smartwatch app bringing technology into OCD treatment](http://www.treatmyocd.com/)<br />
+"tags": ["OCD", "smartphone app", "exposure therapy", "health tech"]<br />
```
Comment (andy5995) [inline Knowledge_Base/01handheld_device_apps.md]: replace "health tech" here with "technology"

Comment (bealearnscode) [reply Knowledge_Base/01handheld_device_apps.md]: Should I remove "smartphone app" as well?

Comment (andy5995) [reply Knowledge_Base/01handheld_device_apps.md]: add "smartwatch app" to the list of tags on this one

Comment (andy5995) [reply Knowledge_Base/01handheld_device_apps.md]: No, I think "smartphone app" is good. Tags unfortunately are not an exact science, lol. In fact I've always had trouble with them. We'll just do our best and make changes far into the future if necessary. :)

**Pull request merged**